symfony/framework-bundle/4.2/src/Kernel.php

```diff
@@ -31,7 +29,7 @@ public function registerBundles()
     {
         $contents = require $this->getProjectDir().'/config/bundles.php';
         foreach ($contents as $class => $envs) {
-            if (($envs['all'] ?? false) && ($envs[$this->environment] ?? true) || ($envs[$this->environment] ?? false)) {
+            if (!empty($envs['all']) || !empty($envs[$this->environment])) {
```

Comment:
this does not do the same as before for all => true, test => false

Reply:
thanks, fixed by using `if ($envs[$this->environment] ?? $envs['all'] ?? false) {`